test/stacktraces.jl

```diff
@@ -0,0 +1,136 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+# Tests for /base/stacktraces.jl
+
+using Base.StackTraces
+using Base.StackTraces: StackFrame, StackTrace, remove_frames!
+
+let
+    @noinline child() = stacktrace()
+    @noinline parent() = child()
+    @noinline grandparent() = parent()
+    line_numbers = [@__LINE__] .- [3, 2, 1]
+
+    @testset "basic" begin
```

Comment:
testsets with the default handler and output level are too noisy for using in base right now